sympy/geometry/line.py

```diff
@@ -2526,6 +2527,49 @@ def equation(self, x='x', y='y', z='z', k=None):
                 break
         return Tuple(*[i.subs(k, kk).as_numer_denom()[0] for i in eqs])
 
+    def bisector(self, line):
```

Comment:
I see that this will return two lines so plural should be used. But perhaps it is better called `loci` and the more general definition I gave could be used even when the lines don't intersect at a point. Bisector should probably be reserved for angles and segments.

Reply:
What you mean by "loc"i is [this](https://en.wikipedia.org/wiki/Locus_(mathematics))?
You are right, it should be plural. However I still do not know how to implement your suggestion of two lines which do not intersect.

Reply:
See [here](https://github.com/sympy/sympy/pull/18358#discussion_r368295351): `line(m, direction_ratio=d)` is one line. The other would be rotated 90 degrees and perpendicular to the same segment, I think.

Reply:
> What you mean by "loc"i is [this](https://en.wikipedia.org/wiki/Locus_(mathematics))?

yes

Reply:
I think I am not able to code that functonality, my math is quite limited. Could you do it, please?